packages/rest/src/router/metadata.ts

```diff
@@ -73,6 +86,20 @@ interface RestEndpoint {
   target: any;
 }
 
+function getEndpoints(
```

Comment:
Can we please have ts docs for this function? (or maybe we don't need it because it's internal?